sections/04_study.md

```diff
@@ -503,49 +502,44 @@ conformation of a protein structure.
 
 Protein contact prediction and contact-assisted folding (i.e. folding proteins
 using predicted contacts as restraints) represents a promising new direction for
-ab initio folding of proteins without good templates in PDB. Evolutionary
-coupling analysis (ECA) is effective for proteins with a very large number
-(>1000) of sequence homologs [@doi:10.1371/journal.pone.0028766], but otherwise
-fares poorly for proteins without many sequence homologs. By combining ECA with
-a few other protein features, shallow neural network-based methods such as
+ab initio folding of proteins without good templates in PDB. Co-evolution analysis 
+is effective for proteins with a very large number (>1000) of sequence 
+homologs [@doi:10.1371/journal.pone.0028766], but otherwise fares poorly for proteins 
+without many sequence homologs. By combining co-evolution information with
+a few other protein features, shallow neural network methods such as
 MetaPSICOV [@doi:10.1093/bioinformatics/btu791] and CoinDCA-NN
-[@doi:10.1093/bioinformatics/btv472] have shown some advantage over ECA for
-proteins with few sequence homologs, but their accuracy is far from optimal. In
-recent years, deeper architectures have been explored for contact prediction.
-For example, Di Lena et al. introduced a deep spatio-temporal neural network (up
-to 100 layers) that utilizes both spatial and temporal features to predict
-protein contacts [@doi:10.1093/bioinformatics/bts475]. Eickholt and Cheng
-combined deep belief networks and boosting techniques to predict protein
-contacts [@doi:10.1093/bioinformatics/bts598] and trained deep networks by
-layer-wise unsupervised learning followed by fine-tuning of the entire network.
-Skwark and Elofsson et al. developed an iterative deep learning technique for
-contact prediction by stacking a series of Random Forests
-[@doi:10.1371/journal.pcbi.1003889]. However, blindly tested in the well-known
-CASP competitions, these methods did not show any advantage over MetaPSICOV
-[@doi:10.1093/bioinformatics/btu791].
-
-Recently, Wang et al. proposed the deep learning method RaptorX-Contact
+[@doi:10.1093/bioinformatics/btv472] have shown some advantage over pure co-evolution analysis for
+proteins with few sequence homologs, but their accuracy is still far from satisfactory. In
+recent years, deeper architectures have been explored for contact prediction, 
+such as CMAPpro [@doi:10.1093/bioinformatics/bts475], DNCON [@doi:10.1093/bioinformatics/bts598]
+and PConsC [@doi:10.1371/journal.pcbi.1003889]. 
+However, blindly tested in the well-known CASP competitions, 
+these methods did not show any advantage over MetaPSICOV [@doi:10.1093/bioinformatics/btu791].
+
+Very recently, Wang et al. proposed the deep learning method RaptorX-Contact
```

Comment:
It's not too important to me, but I suggested removed "Very" during our last review.  Someone reading this in 2018 (or later) may think a method that was pre-printed in 2016 isn't "very" recent any more.  Do you care if we remove it?